README.md

```diff
@@ -790,6 +790,24 @@ May also be configured in `lerna.json`:
 }
 ```
 
+#### --npm-client-args [client]
+
+Pass extra arguments to npm client.
+
+```sh
+$ lerna bootstrap --npm-client=yarn --npm-client-args=--ignore-engine
```

Comment:
As `--npm-client-args` is just a string and simply proxies everything to underlying npm client, it'll be:

```
lerna bootstrap --npm-client=yarn --npm-client-args='--ignore-engine --flag1 --flag2 ...'
```